docs/installation/installation_kubernetes_gcloud.md

```diff
@@ -0,0 +1,56 @@
+# Deploying Susi with Kubernetes on Google Cloud
+
+
+1. Go to [Google Cloud Free Trial](https://cloud.google.com/free-trial/) and sign up. You will get 300 dollars credit for 3 months.
+
+
+2. Go to the console page: [Console](https://console.cloud.google.com/home)
+
+
+3. If you don’t have any projects, a page that would guide you on how to start a project will pop up. Give your project a name.
+
+
+4. In the search bar above on console page, enter 'Compute Engine' and click on ```Compute Engine```. 
+
+
+5. If you have not activated billing for compute engine, click enable billing and choose your account to activate your Compute Engine. Your Google Compute Engine should be activated within a few minutes.
+
+
+6. Click ```Create instance``` to create an instance. In machine type, choose ```small(1 shared vCPU)```.
+	Choose whatever zone you like.
+ 
+	In Boot Disk, choose ```Ubuntu 14.04```, and the disk size should be larger than 40GB.
+	On the bottom of the page, click ```Management, disk, networking, access & security options``` to show more options. Inside of this, click into the tab ```Networking```.
+	Check the two boxes ```Allow HTTP traffic``` and ```Allow HTTPS traffic```. Finally, click ```Create``` to create an instance. Wait a few minutes for the creation to complete.
+
+
+7. Once the creation has finished, click ssh below to establish web ssh connections.
+
+8. In the Web Console, enter:
```

Comment:
Add the information about console that open google cloud shell from icon besides profile avatar on right top corner. and mention that commands are needed to be input in shell.